packages/block-library/src/navigation/edit/use-create-navigation-menu.js

```diff
@@ -86,5 +86,9 @@ export default function useCreateNavigationMenu( clientId ) {
 		status,
 		value,
 		error,
+		isIdle: status === CREATE_NAVIGATION_MENU_IDLE,
+		isPending: status === CREATE_NAVIGATION_MENU_PENDING,
+		didSucceed: status === CREATE_NAVIGATION_MENU_SUCCESS,
+		didError: status === CREATE_NAVIGATION_MENU_ERROR,
```

Comment:
```suggestion
		isSuccess: status === CREATE_NAVIGATION_MENU_SUCCESS,
		isError: status === CREATE_NAVIGATION_MENU_ERROR,
```

@talldan I wanted to use the `is` prefix so I've gone with renaming here.